bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/template/TemplateParametersEditor.scala

```diff
@@ -65,7 +65,7 @@ object TemplateParametersEditor {
     }
   }
 
-  abstract class ColumnPanel(hGap: Int = HGap, rowAnchor: GridBagPanel.Anchor.Value = East) extends GridBagPanel with Initializable {
+  sealed abstract class ColumnPanel(hGap: Int = HGap, rowAnchor: GridBagPanel.Anchor.Value = East) extends GridBagPanel with Initializable {
```

Comment:
Do you need `sealed` if the trait is already `sealed`?

Reply:
No, probably not. I may have gotten a little `seal`-happy. I'll go ahead and remove it since it's redundant.